src/statd/python/yanger/yanger.py

```diff
@@ -466,12 +457,14 @@ def get_bridge_port_stp_state(ifname):
     return None
 
 
-def container_inspect(name, key):
-    """Call podman inspect {name}, return object at {path} or None"""
+def container_inspect(name):
+    """Call podman inspect {name}, return object at {path} or None."""
     cmd = ['podman', 'inspect', name]
-    raw = run_json_cmd(cmd, "")
-
-    return getitem(raw[0], key)
+    try:
+        return run_json_cmd(cmd, "")
+    except Exception as e:
+        logging.error(f"Error running podman inspect: {e}")
+        return None
```

Comment:
```suggestion
        return {}
```
Right? Otherwise you will b0rk in `add_container()`